CONTRIBUTING.md

```diff
@@ -0,0 +1,78 @@
+   We do not enforce a naming convention for the PRs, but **please use something descriptive of your changes**, having in mind that the title of your PR will be automatically added to the next [release changelog](https://github.com/meilisearch/meilisearch-migration/releases/).
+
+## Git Guidelines
+
+### Git Branches <!-- omit in TOC -->
+
+All changes must be made in a branch and submitted as PR.
+We do not enforce any branch naming style, but please use something descriptive of your changes.
+
+### Git Commits <!-- omit in TOC -->
+
+As minimal requirements, your commit message should:
+
+- be capitalized
+- not finish by a dot or any other punctuation character (!,?)
+- start with a verb so that we can read your commit message this way: "This commit will ...", where "..." is the commit message.
+  e.g.: "Fix the home page button" or "Add more tests for create_index method"
+
+We don't follow any other convention, but if you want to use one, we recommend [this one](https://chris.beams.io/posts/git-commit/).
+
+### GitHub Pull Requests <!-- omit in TOC -->
+
+Some notes on GitHub PRs:
+
+- [Convert your PR as a draft](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/changing-the-stage-of-a-pull-request) if your changes are a work in progress: no one will review it until you pass your PR as ready for review.<br>
+  The draft PR can be very useful if you want to show that you are working on something and make your work visible.
+- The branch related to the PR must be **up-to-date with `main`** before merging. If it's not, you have to rebase your branch. Check out this [quick tutorial](https://gist.github.com/curquiza/5f7ce615f85331f083cd467fc4e19398) to successfully apply the rebase from a forked repository.
+- All PRs must be reviewed and approved by at least one maintainer.
+- The PR title should be accurate and descriptive of the changes.
+
+## Release Process (for internal team only)
+
+The release tags of this package follow exactly the MeiliSearch versions.<br>
+It means that, for example, the `v0.17.0` tag in this repository corresponds to the scripts for deploying MeiliSearch `v0.17.0`.
+
+This repository currently does not provide any automated way to test and release the cloud scripts.<br>
+**Please, follow carefully the steps in the next sections before any release.**
+
+### Release <!-- omit in TOC -->
+
+1.  Create a PR pointing to `main` branch and merge it.
+
+2.  Move the tag to the last commit of the `main` branch.
+
+```bash
+$ git tag -d vX.X.X
+$ git push --delete origin vX.X.X
+$ git checkout main
+$ git pull origin main
+$ git tag vX.X.X
+$ git push origin vX.X.X
+```
```

Comment:
No need to do command line command since you add the release drafter and you will publish the release via GitHub

Reply:
I was wondering if we wanted the release drafter! I think its a good idea :) I will change the contributing